Doc/library/wsgiref.rst

```diff
@@ -347,6 +347,14 @@ request.  (E.g., using the :func:`shift_path_info` function from
    :func:`make_server` function.  Some possibly relevant methods for overriding in
    subclasses:
 
+   :class:`WSGIRequestHandler` has the following class variables:
+
+   .. attribute:: server_handler
+
+      .. versionadded:: 3.9
+
+      The handler class to handle the request. This value should be the subclass
+      :class:`wsgiref.handlers.BaseHandler`
```

Comment:
I don't know we should expose this attribute or not. For me, the `handle()` method has already document the behaviour very well.